### ISSUE TITLE
MainWindowController does not respond to additionalInfoTitle

### ISSUE THREAD

**Issue by low-batt** [iina/iina#3811]
**System and IINA version:**

- macOS 12.4
- IINA 1.3.0

**Expected behavior:**
IINA runs under Xcode without emitting messages indicating the code is not correct.

**Actual behavior:**
The following message is emitted in the Xcode console:
```text
15:04:41.595 [player0][d] Opening /Users/low-batt/Movies/Airplaneski.mp4 in main window
2022-06-08 15:04:41.612687-0400 IINA[54039:482704] [Nib Loading] Could not connect action, target class IINA.MainWindowController does not respond to -additionalInfoTitle:
```

This is not new to 1.3.0. I have not seen any bad behavior due to this issue. But this should be cleaned up at some point.

**Steps to reproduce:**
Run IINA in Xcode, play a video, look for this message in the Xcode console.

- [x] MPV does not have this problem.

**How often does this happen?**
Every time.

**Comment by low-batt** [comment]
The commit in the pull request will remove an extraneous and unneeded action connection between the XIB and `MainWindowController` for the `additionalInfoTitle` label.

**Comment by low-batt** [comment]
The fix in PR [#3830](https://github.com/iina/iina/pull/3930) has been merged into the develop branch.

**Comment by low-batt** [comment]
This is fixed in the released version of IINA.
